src/Apps/Collect/Components/Collection/Header/index.tsx

```diff
@@ -68,142 +88,165 @@ const imageWidthSizes = {
   xl: 1112,
 }
 
-@track({
+track({
```

Comment:
I wonder if you could decorate functions in Javascript. Have you tested tracking in the actual browser?

Reply:
Yeah the tracking logic is currently not working which is known. There is a PR that's close to being merged that introduces hook-friendly track pattern.

Reply:
I have one last follow-up PR that is about to be merged so this can continue on Monday.

Reply:
Once https://github.com/artsy/reaction/pull/2434 is merged, should be able to pull `trackEvent` out via a hook: 

Pseudocode: 
```jsx
import { useTracking } from 'Artsy/Analytics'

const { trackEvent } = useTracking()

const trackClick = () => {
  trackEvent({
     ... 
  })
}
<div onClick={() => trackClick()} />
```